gno.land/pkg/integration/pkgloader.go

```diff
@@ -39,7 +40,7 @@ func (pl *PkgsLoader) SetPatch(replace, with string) {
 	pl.patchs[replace] = with
 }
 
-func (pl *PkgsLoader) LoadPackages(creator bft.Address, fee std.Fee, deposit std.Coins) ([]gnoland.TxWithMetadata, error) {
+func (pl *PkgsLoader) LoadPackages(creator bft.Address, fee std.Fee, deposit std.Coins, creatorKey crypto.PrivKey) ([]gnoland.TxWithMetadata, error) {
```

Comment:
We can drop the `bft.Address` param, since you can get the address from the `creatorKey`

Reply:
donde on e0bb164e1a305aa2f130131fc98b692ac8644312